en/micro-integrator/docs/references/connectors/salesforce-soap-connector/salesforce-soap-reference.md

```diff
@@ -0,0 +1,1121 @@
+
+---
+
+## Initialize the connector
+
+To use the Salesforce SOAP connector, add the `<salesforcerest.init>` element in your configuration before carrying out any other Salesforce SOAP operations. 
+
+??? note "salesforcebulk.init"
+    The salesforcerest.init operation initializes the connector to interact with the Salesforce SOAP API. See the [related API documentation](https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_quickstart_intro.htm) for more information.
+    <table>
+        <tr>
+            <th>Parameter Name</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+        <tr>
+            <td>username</td>
+            <td>The username to access the Salesforce account.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>password</td>
+            <td>The password provided here is a concatenation of the user password and the security token provided by Salesforce.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>loginUrl</td>
+            <td>The login URL to access the Salesforce account.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>blocking</td>
+            <td>Indicates whether the connector needs to perform blocking invocations to Salesforce. (Supported in WSO2 ESB 4.9.0 and later.)</td>
+            <td>Yes</td>
+        </tr>
+    </table>
+
+    **Sample configuration**
+
+    ```xml
+    <salesforce.init>
+        <loginUrl>{$ctx:loginUrl}</loginUrl>
+        <username>{$ctx:username}</username>
+        <password>{$ctx:password}</password>
+        <blocking>{$ctx:blocking}</blocking>
+     </salesforce.init>
+    ```
+
+    **Sample request**
+
+    ```xml
+    <salesforce.init>
+        <username>MyUsername</username>
+        <password>MyPassword</password>
+        <loginUrl>https://login.salesforce.com/services/Soap/u/42.0</loginUrl>
+        <blocking>false</blocking>
+    </salesforce.init>
+    ```
+---
+
+## Working with emails
+    
+??? note "emails"
+    The salesforcebulk.emails method creates and sends an email using Salesforce based on the properties that you specify. See the [related API documentation](https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_calls_sendemail.htm) for more information.
+    <table>
+        <tr>
+            <th>Parameter Name</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+        <tr>
+            <td>sendEmail</td>
+            <td>XML representation of the email.</td>
+            <td>Yes</td>
+        </tr>
+    </table>
+
+    **Sample configuration**
+    
+    ```xml
+    <salesforce.sendEmail>
+        <sendEmail xmlns:sfdc="sfdc">{//sfdc:emailWrapper}</sendEmail>
+    </salesforce.sendEmail>
+    ```
+
+    **Sample request**
+    
+    Given below is a sample request that can be handled by the sendEmail operation.
```

Comment:
what is given below is not a request. It is the proxy itself. The request body is constructed within the proxy itself. Same comment for other operations as well

Reply:
I refereed the this reference doc content from git repo (https://github.com/wso2-extensions/esb-connector-salesforce/blob/master/connector/docs/records.md#creating-records). Could you please suggest correct heading for this.

Reply:
We can say "Given below is part of the proxy that can be invoked to test sendEmail operation with hardcoded request"